src/imagekit.cs

```diff
@@ -199,19 +197,17 @@ interface IKDeviceBrowserView {
 		[Export ("mode")]
 		IKDeviceBrowserViewDisplayMode Mode { get; set;  }
 
-//		FIXME need ImageCaptureCore;
-//		[Export ("selectedDevice")]
-//		ICDevice SelectedDevice { get;  }
+		[Export ("selectedDevice")]
+		ICDevice SelectedDevice { get;  }
 	}
 
 	[BaseType (typeof (NSObject))]
 	[Model]
 	[Protocol]
 	interface IKDeviceBrowserViewDelegate {
-//		FIXME need ImageCaptureCore;
-//		[Abstract]
-//		[Export ("deviceBrowserView:selectionDidChange:"), EventArgs ("IKDeviceBrowserViewICDevice")]
-//		void SelectionDidChange (IKDeviceBrowserView deviceBrowserView, ICDevice device);
+		[Abstract]
```

Comment:
Can't add `[Abstract]` members in an existing interface, since it's a breaking change. Do this instead:

```csharp
#if XAMCORE_4_0
    [Abstract]
#endif
```

so that we get it right next time we can break compatibility.

Reply:
I would tend to agree, but this API is basically broken, it served no use without this method, which is the root of this request, I think this is ok.